lib/typescript/websocketclient/index.ts

```diff
@@ -37,34 +37,30 @@ export class WebSocketClient {
     this.stompWrapper.initConnection();
   }
 
-  destroyConnection = () => {
-    this.stompWrapper.destroyConnection();
-  };
+  destroyConnection = () => this.stompWrapper.destroyConnection();
 
   onEvent = (body: string) => {
     const json = JSON.parse(body) as EventPayload;
     switch (json.type) {
-      case 'channel':
+      case 'channel.updated':
```

Comment:
@AudreyKj and @thorstenairy The websocket event names will change a bit in the next release.